build/dt-report-generator-bundle.js

```diff
@@ -7,13 +7,35 @@
 
 const browserify = require('browserify');
 const fs = require('fs');
+const rimraf = require('rimraf');
 
-const distDir = __dirname + '/../dist';
-const outFile = `${distDir}/report-generator.js`;
+const distDir = __dirname + '/../dist/dt-resources';
+const bundleOutFile = `${distDir}/report-generator.js`;
 const generatorFilename = `./lighthouse-core/report/report-generator.js`;
-browserify(generatorFilename, {standalone: 'ReportGenerator'})
-  .transform('brfs')
+const htmlReportAssets = require('../lighthouse-core/report/html/html-report-assets.js');
+
+/**
+ * @param {string} name
+ * @param {string} content
+ */
+function cachedResource(name, content) {
```

Comment:
it looks like this might be the ascii thing you were talking about? let's give the function a bit more descriptive name then

`convertToAsciiAndWriteFile`, is that what it's doing?